src/sdk/account/classes/account-investment-positions-info.ts

```diff
@@ -0,0 +1,19 @@
+import { BigNumber } from 'ethers';
+import { TransformBigNumber } from '../../common';
+
+export class AccountInvestmentPositionsInfo {
+    label: string;
```

Comment:
What do the fields label, metaType represent?

Reply:
It is a name

Reply:
The reason to keep both name and label as Zapper returns a different value 

```
 {
                "label": "Claimable AAVE",
                "balance": "0x2eb167f5f0e794",
                "tokens": 0.00023638325529935,
                "price": 55.6,
                "metaType": "claimable",
                "name": "AAVE",
                "logoURI": "https://storage.googleapis.com/zapper-fi-assets/tokens/ethereum/0x7fc66500c84a76ad7e9c93437bfc5ac33e2ddae9.png"
              
}
```

Reply:
![13-Gwei-Zapper-Your-Home-to-Web3](https://user-images.githubusercontent.com/78725072/214007077-c6800c56-82c2-4b5d-8590-1d33c853246c.png)

Reply:
But others wont support this, no? Is this used anywhere within our dashboard or app?

Reply:
@ch4r10t33r In our app, we are not showing that label. We are sending multiple arrays in the API. So, if we try to display it like a zapper in the future, the label will be useful to distinguish

I would agree, other providers might not have. I can remove it if it is unnecessary for now. What is your thought? If yes, I will remove it from the API too.

Reply:
As this field is specific to zapper, I suggest we remove it as other providers might not support it.

Reply:
@ch4r10t33r Makes sense. Removed.